papis/tui/utils.py

```diff
@@ -6,6 +6,7 @@
 # Highlighting style used by pygments. This is a copy of the prompt_toolkit
 # default style, but changed to use ansi colors.
 PAPIS_PYGMENTS_DEFAULT_STYLE = {
+    "": "bg:ansiblack",
```

Comment:
@jghauser I haven't looked at how the `prompt_toolkit` style stuff works, so I haven't managed to find a nicer way to make the background of the whole `print_formatted_text` in `text_area` black. Is there a better way to add some styling to all the elements?

Reply:
One worry is that it might cause problems when the default background color of the terminal is set to something light, but I haven't actually checked yet. I'll look into it -- I have to admit I mostly made things work by trial and error :sweat_smile:

Reply:
Hm, the diff widget should have the same problem. I was mostly looking for a way to have that chunk of text be emphasized a bit, so maybe there are better ways..

Reply:
Removed this.. definitely looked weird on a white background :(